### PR TITLE
*: minor fixes

### PR DESCRIPTION
**Pull request by jonboulle**

### CODE REVIEW

**Comment by jonboulle** [comment]
PING YES HELLO :-)

**Comment by philips** [comment]
LGTM :)

**Comment by jonboulle** [comment]
what are you doing here..

**Comment by philips** [comment]
I've been here the whole time, Jon.

File: tar/storage/packer.go [hunk 1]
```diff
@@ -90,11 +90,7 @@ type jsonPacker struct {
 	seen seenNames
 }
 
-type seenNames map[string]byte
-
-// used in the seenNames map. byte is a uint8, and we'll re-use the same one
-// for minimalism.
-const emptyByte byte = 0
```
Comment (vbatts) [inline tar/storage/packer.go]: why `struct{}{}` instead of a tiny integer?

**Comment by vbatts** [comment]
/me curses github notifications

@jonboulle could you rebase?

File: tar/storage/packer.go (hunk 1 above)
Comment (jonboulle) [inline tar/storage/packer.go]: more idiomatic for sets (thought this was in effective go but can't seem to find it now - but http://dave.cheney.net/2014/03/25/the-empty-struct )

Comment (vbatts) [inline tar/storage/packer.go]: ah. thanks.

**Pull request closed without merging**